action-allowedlist/approved.json

```diff
@@ -1,243 +1,131 @@
 [
-  {
-    "actionLink": "actions/checkout",
-    "actionVersion": "5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f",
-    "tag": "v2.3.4",
-    "deprecated": true
-  },
-  {
-    "actionLink": "actions/checkout",
-    "actionVersion": "ec3a7ce113134d7a93b817d10a8272cb61118579",
-    "tag": "v2.4.0",
-    "deprecated": true
-  },
-  {
-    "actionLink": "actions/checkout",
-    "actionVersion": "a12a3943b4bdde767164f792f33f40b04645d846",
-    "tag": "v3.0.0",
-    "deprecated": true
-  },
-  {
-    "actionLink": "actions/checkout",
-    "actionVersion": "2541b1294d2704b0964813337f33b291d3f8596b",
-    "tag": "v3.0.2",
-    "deprecated": true
-  },
-  {
-    "actionLink": "actions/checkout",
-    "actionVersion": "755da8c3cf115ac066823e79a1e1788f8940201b",
-    "tag": "v3.2.0",
-    "deprecated": true
-  },
-  {
-    "actionLink": "actions/checkout",
-    "actionVersion": "ac593985615ec2ede58e132d2e21d2b1cbd6127c",
-    "tag": "v3.3.0",
-    "deprecated": true
-  },
   {
     "actionLink": "actions/checkout",
     "actionVersion": "8ade135a41bc03ea155e62e844d188df1ea18608",
-    "tag": "v4.1.0"
+    "tag": "v4.1.0",
+    "deprecated": true
```

Comment:
Encouraging - but not requiring - an update because there is a bug fix available (below) and a newer version with a feature update.